bryant-bryantbuilt/week-2/js-racer-game-lab/starter-code/app.js

```diff
@@ -2,31 +2,28 @@ window.addEventListener("keydown", function(e){
   var pressedButton = e.keyCode;
```

Comment:
Just a recommendation, but if you are going to use ES5 or ES6 syntax in your work it is standard practice to stick to one for your entire file/project. So if you are using var use it everywhere the same with let.